ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_security.schema.yml

```
@@ -154,3 +154,10 @@ keys:
                   type: str
                 key:
                   type: str
+            certificate_revocation_lists:
+              type: list
+              description: |
+                List of CRLs (Certificate Revocation List).
+                One CRL needs to be specified for every CA in the chain, even if it's not revoking any certificate.
```

Not sure I understand what this means exactly... There should be only one CA right and then intermatidate CAs maybe? so a config is only valid if there is an entry for every single intermediate CA?
```suggestion
                One CRL needs to be specified for every CA in the chain, even if it is not revoking any certificate.
```

---

This means every intermediate certificate as well as root certificate.

If a CRL for each is not specified, the SSL profile won't be valid.

---

Empty does not mean that the crl file can be empty. It needs to be a fully formed CRL in PEM format, just that the revoked certs list in the CRL can be empty.